Lannister/App/Controllers/DashboardController.swift

```diff
@@ -113,23 +114,35 @@ class DashboardController: UIViewController {
             view.sendSubviewToBack(emptyStateContainerView)
             collectionView.isUserInteractionEnabled = true
             
-            if sortKey == "amount" {
-                self.holdings = holdings.sorted { Currencies.getEuroValue(value: $0.value, currency: $0.currency)/self.euroTotalValue*100 > Currencies.getEuroValue(value: $1.value, currency: $1.currency)/self.euroTotalValue*100 }
-            } else {
-                self.holdings = holdings.sorted { $0.name! < $1.name! }
-            }
+//            if sortKey == "amount" {
```

Comment:
If we're not using this code anymore can we remove it?

Reply:
No :) we need an additional computed property (for example total_value) to associate to `Holding` entity, so we can use it for sorting holdings by amount.

Reply:
But it's not doing anything right now right?

Reply:
Right, but we lost the sorting ability by amount. Need to fix it and put it back in.